pyresample/test/test_spherical.py

```diff
@@ -605,6 +656,74 @@ def test_intersection(self):
         self.assertTrue(np.allclose(poly_inter.vertices,
                                     np.deg2rad(res)))
 
+        # Test when intersection occurs across the antimeridian
+        vertices_epsg_4326 = np.deg2rad(np.array([[-180, -90],
+                                                  [-180, 90],
+                                                  [0, 90],
+                                                  [180, 90],
+                                                  [180, -90]]))
+        vertices_goes_west = np.array([[2.50919361e+00, 1.19782309e-16],
```

Comment:
Do we need *all* these points for this test to work and make sense? Meaning, does it fail before your changes if it only had 5 or 6 points?

Reply:
Do you mean if I can simply the goes-west vertices @djhoese ?

Reply:
Yes. Can it use less vertices?

Reply:
Done